lib/spaceship/tunes/tunes_client.rb

```diff
@@ -1,5 +1,10 @@
 module Spaceship
   class TunesClient < Spaceship::Client
+    class << self
+      attr_accessor :login_cache_path
+    end
+    TunesClient.login_cache_path = "/tmp/spaceship_itc_login_url.txt"
```

Comment:
Wait, why is that even its own attribute? It never changes, this should be a constant.